### PR TITLE
[OFFAPPS-666] updated translations

### PR DESCRIPTION
**Pull request by pmgrove**
:v: :

Updated translations

/cc @zendesk/vegemite @ydrain 

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-666

### Risks
 - None

### CODE REVIEW

File: translations/bg.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "app": {
+    "description": "Displays customer, order, and subscription details from Infusionsoft",
```
Comment (adammw) [inline translations/bg.json]: Is this supposed to be English?

**Comment by adammw** [comment]
:+1: although out of curiosity, where are you getting these translations from?

**Comment by sandlerr** [comment]
`zat translate update` ?

File: translations/bg.json (hunk 1 above)
Comment (sandlerr) [inline translations/bg.json]: that's just how zat works. This locale was created in our DB but not much has been localised in it yet.

Comment (ydrain) [inline translations/bg.json]: Bulgarian is managed by a crowdsourced translator, and they haven't provided the translations.

**Comment by ydrain** [comment]
:+1: 

cc: @injuhwang

**Pull request merged**